application/server/models/character.go

```diff
@@ -12,8 +12,8 @@ type Character struct {
 	ClassType   uint
 	OwnerID     uint
 	Owner       User    `gorm:"foreignkey:OwnerID"`
-	Spells      []Spell `gorm:"many2many:character_spells;"`
-	Items       []Item  `gorm:"many2many:character_items;"`
```

Comment:
was this intentional? The docs have ; at the end

Reply:
I think it's ok but i'll put it back so nothing breaks